encodec.cpp

```diff
@@ -151,11 +151,11 @@ static struct ggml_tensor * strided_conv_transpose_1d(
     return unpadded;
 }
 
-bool encodec_model_load(const std::string& fname, encodec_model& model) {
+int encodec_model_load(const std::string& fname, encodec_model& model) {
```

Comment:
c has the bool type

https://en.cppreference.com/w/c/language/arithmetic_types#Boolean_type